android/app-project-newm/src/main/java/io/projectnewm/screens/login/LoginScreen.kt

```diff
@@ -0,0 +1,200 @@
+package io.projectnewm.screens.login
+
+import androidx.annotation.DrawableRes
+import androidx.compose.foundation.*
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Button
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Text
+import androidx.compose.runtime.*
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.focus.FocusRequester
+import androidx.compose.ui.focus.focusRequester
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.SolidColor
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.semantics.Role.Companion.Button
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.FontStyle
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.text.style.TextDecoration
+import androidx.compose.ui.unit.dp
+import io.projectnewm.R
+import io.projectnewm.components.ArtistRingBrush
+import io.projectnewm.components.SongRingBrush
+import io.projectnewm.screens.login.email.Email
+import io.projectnewm.screens.login.email.EmailState
+import io.projectnewm.screens.login.password.Password
+import io.projectnewm.screens.login.password.PasswordState
+import io.projectnewm.utilities.ToBeImplemented
+import io.projectnewm.utilities.shortToast
+
+@Composable
+fun LoginScreen(
+    onSignInSubmitted: (email: String, password: String) -> Unit,
+    scrollableState: ScrollState = rememberScrollState()
+) {
+    val context = LocalContext.current
+
+    LoginPageBackgroundImage(backgroundImage = R.drawable.login_background_1)
+    Column(
+        modifier = Modifier
+            .fillMaxWidth()
+            .padding(horizontal = 30.dp)
+            .verticalScroll(scrollableState),
+        horizontalAlignment = Alignment.CenterHorizontally,
+        verticalArrangement = Arrangement.Center
+    ) {
+
+        Spacer(modifier = Modifier.height(50.dp))
+
+        LoginPageMainImage(R.drawable.ic_newm_logo)
+        LoginPageMainTextImage(textImage = R.drawable.login_enter_newminiverse)
```

Comment:
It's not letting me comment on the icon file so I'll comment here:
Let's prefix icons with ic_ and backgrounds with bg_

Also typo `newminiverse` -> `newmiverse`